python/src/ops.cpp

```diff
@@ -2150,6 +2150,38 @@ void init_ops(py::module_& m) {
         Returns:
             array: The concatenated array.
       )pbdoc");
+  m.def(
+      "stack",
+      [](const std::vector<array>& arrays,
+         std::optional<int> axis,
+         StreamOrDevice s) {
+        if (axis.has_value()) {
+          return stack(arrays, axis.value(), s);
+        } else {
+          return stack(arrays, s);
+        }
+      },
+      "arrays"_a,
+      py::pos_only(),
+      "axis"_a = 0,
+      py::kw_only(),
+      "stream"_a = none,
+      R"pbdoc(
+      stack(arrays: List[array], axis: Optional[int] = 0, *, stream:
+      Union[None, Stream, Device] = None) -> array
```

Comment:
Same as below style change, could you put it all in one line to match the rest of the signatures? Maybe in the future we 'll enforce all of them in 80 or 120 chars but for now...